map-of-pi/map-of-pi-backend-react
# ft: user authentication

## what doe sthis PR does
in this Pr i have added basic(initial user authentciation ) allowing to authenticate usr from fronted to backend as well

## how to test this
 to test this first copy env.exmple and put in your env
checkout to thsi baranc and run dev server
make sure u have fronted running in sandbox
chnage port accordingly either fronted or backend

once u are running app in sandbox u u will see auth screen forn pi sdk and if u chekc on backend in terminal u will see user data being logged
